### ISSUE TITLE
messenger image upload fails (in various ways)

### ISSUE THREAD

**Issue by StevanWhite** [rignaneseleo/SlimSocial-for-Facebook#198]
This is version 10.0.9 on Android 10.  Plenty of RAM.  On a fast, reliable WI-FI.

In messenger, especially:
Upload a file from photo library.

When it works, it pauses at the end, and then, pretty much every time, says that a problem had occurred, and the images was not transmitted.
Yet, often, the image *does* appear in the chat window.

Often it does not work at all.  The usual symptom is, the app just freezes after the image is chosen.

Sometimes I have got it going again by turning the app off and deleting the cache.

It would make your app SO much more useful, if you could correct this!

**Comment by test2a** [comment]
Yeah. For me some images don't show any size so they get stuck on 0KB/0KB...

Tried to upload muliple times but nothing.

**Comment by rignaneseleo** [comment]
Thank you for the report, and I am sorry about the long delay.

This report is against an old version. The app is in active development again since August 2026, and much of it changed: the user agent, a new "Use desktop site" setting, the feed filters, the Messenger screen and photo upload. Old reports are no longer a good reference.

Note: Image upload inside Messenger on v10.0.9. Photo upload was fixed in 26.08.29+125, and 26.09.02+127 added multi-file selection.

I close this issue to clean up the tracker. Please install **26.09.03+128** or later, from the Releases page, the Play Store or F-Droid. If the problem is still there, open a new issue with your device model, your Android version and the app version.
